docs/src/using-turing/advanced.md

```diff
@@ -191,5 +191,4 @@ model = Turing.Model(modelf, (x = [1.5, 2.0],))
 
 ## Task Copying
 
-
-Turing [copies](https://github.com/JuliaLang/julia/issues/4085) Julia tasks to deliver efficient inference algorithms, but it also provides alternative slower implementation as a fallback. Task copying is enabled by default. Task copying requires us to use the `CTask` facility which is provided by [Libtask](https://github.com/TuringLang/Libtask.jl) to create tasks.
+Turing [copies](https://github.com/JuliaLang/julia/issues/4085) Julia tasks to deliver efficient inference algorithms, but it also provides alternative slower implementation as a fallback. Task copying is enabled by default. Task copying requires us to use the `TapedTask` facility which is provided by [Libtask](https://github.com/TuringLang/Libtask.jl) to create tasks.
```

Comment:
It looks good. By the way, soon we can also remove the `TArray` related docs since the new `Libtask` can deep-copy heap-allocated objects - we still need to activate it. See https://github.com/TuringLang/Libtask.jl/blob/4a85b47756259c2778c4336c69b7abc0b61a0fb9/src/tapedfunction.jl#L309